### ISSUE TITLE
Get stuck when changing from iMacPro1,1 to iMac20,2

### ISSUE THREAD

**Issue by scyu6058** [SchmockLord/Gigabyte-Z590i-Vision-D-11900k#27]
I have almost the same configurations except that I used internal AMD display card.  Everything works fine when using the provided EFI.   But when I attempted to change the SMBIOS to iMac20,2, the system stopped at the log-in page with mouse and keyboard all frozen up.  Any idea, please.

**Comment by DarioBogus** [comment]
Hi there.
Did you disabled the integrated GPU in the BIOS or you leave it "automatic"?

**Comment by scyu6058** [comment]
But seems the USB Mapping kext  was built specifically for iMacPro1,1.   Anyway, I'll try as highlighted.

**Comment by DarioBogus** [comment]
Surely USB Mapping on iMacPro1,1 SMBIOS needs the XHC1 renamd to SHCI, but this should not interfere with system startup. 
You can try with the USBinjectall.kext though, without custom USB mapping.

**Comment by SchmockLord** [comment]
Yep. The USBMap.kext is specific for iMacPro1,1. Open Package contents, search for the info.plist and then search for "iMacPro1,1" and replace it by "iMac20,2".

But why do you want to use iMac20,2? I would only use it if you have a supported iGPU. But the 11900k has no supported iGPU.

**Comment by scyu6058** [comment]
Yep. The USBMap.kext is specific for iMacPro1,1. Open Package contents, search for the info.plist and then search for "iMacPro1,1" and replace it by "iMac20,2".

But why do you want to use iMac20,2? I would only use it if you have a supported iGPU. But the 11900k has no supported iGPU.

**Comment by scyu6058** [comment]
> 

Thanks for the enlightenment.
